rust_icu_ecma402/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The list formatter uses features that are not available in ICU versions prior
+// to 67.
```

Comment:
What's the context for this comment? Should there be a `#[cfg(icu_version_67_plus)]` here?

Reply:
There used to be one, but was removed in favor of a partial implementation.

Reply:
So do we want this information to be visible somewhere? Using `//!` as a module comment, or in the README?